### PR TITLE
Replaces the roman shield in the autodrobe with a replica one.

### PR DESCRIPTION
**Pull request by Spacedong**
The roman shield once found in the autodrobe is now a replica made of cheap plastic, meaning it will only block some melee hits if you're lucky, and won't block any ranged sources of damage. Skeletons still get the real deal however. Fixes issue #3039

### CODE REVIEW

**Comment by Hackole** [comment]
+1

File: code/game/objects/items/weapons/shields.dm [hunk 1]
```diff
@@ -21,6 +21,22 @@
 	origin_tech = "materials=2"
 	block_chance = list(melee = 60, bullet = 50, laser = 50, energy = 50) //Skeleton-exclusive atm, sooo
 
+/obj/item/weapon/shield/roman/replica
+	name = "roman shield"
+	desc = "Bears an inscription on the inside: <i>\Romanes venio domus\"</i>. Feels flimsy."
+	icon_state = "roman_shield"
+	item_state = "roman_shield
```
Comment (JohnGinnane) [inline code/game/objects/items/weapons/shields.dm]: Missing end quote, inb4 travis

**Comment by JohnGinnane** [comment]
I'm faster than Travis :¬)

**Comment by trifornt-tinfornt** [comment]
do it do it
DO IT

File: code/game/objects/items/weapons/shields.dm [hunk 2]
```diff
@@ -21,6 +21,22 @@
...
+	icon_state = "roman_shield"
+	item_state = "roman_shield"
+	slot_flags = SLOT_BACK
+	force = 8
+	stamina_percentage = 0.75
+	throwforce = 5
+	throw_speed = 2
+	throw_range = 3
+	w_class = 4
+	materials = list(MAT_GLASS=7500, MAT_METAL=1000)
```
Comment (JohnGinnane) [inline code/game/objects/items/weapons/shields.dm]: If this is a replica, why is it made up of the same materials?

Comment (Spacedong) [inline code/game/objects/items/weapons/shields.dm]: you're welcome to port plastic from /vg/ or baystation my man :wink:

File: code/game/objects/items/weapons/shields.dm [hunk 3]
```diff
@@ -21,6 +21,22 @@
 	origin_tech = "materials=2"
 	block_chance = list(melee = 60, bullet = 50, laser = 50, energy = 50) //Skeleton-exclusive atm, sooo
 
+/obj/item/weapon/shield/roman/replica
```
Comment (JohnGinnane) [inline code/game/objects/items/weapons/shields.dm:24]: A lot of the definitions you have here (e.g. name, desc, etc.) are the same as the parent type (`/obj/item/weapon/shield/roman`), so you don't need to define them again as it'll inherit the parent's values.

Also perhaps we should name it "replicate roman shield"?

**Comment by ArcLumin** [comment]
>fake plastic shield

>only difference is damage blocked with no other changes to shield stats

DONGCODE BOYS

**Comment by LufgtHuron** [comment]
this isnt needed, I mean what the fuck is the point? are we going for realism in a place where you can fart and kill someone with an explosion?

**Comment by JohnGinnane** [comment]
It's not about going for "muh realism", it's a balance change. Powergamers could rush it and be very strong, and it wasn't difficult to get

**Comment by ArcLumin** [comment]
I'm not meaning the muh realism part, but the stam blocking and same attacking force

**Comment by ArcLumin** [comment]
can you not fuckin count +1's specco donk

**Comment by McBawbaggings** [comment]
Yeah okay but I was working on my own PR that did this.

**Comment by blobbernaut** [comment]
this is such a heavy nerf you should change the sprite and make it smaller or just remove it altogether

Make it like, 10% for all the ballistics, should be fine.

**Comment by McBawbaggings** [comment]
This was an oversight that never got fixed.
Make a tray shield if you want a shield.

**Comment by JohnGinnane** [comment]
Code's good

**Comment by McBawbaggings** [comment]
This will cause conflicts with my PR, if Hackole would rather have 0% block chance or 20% on melee then we'll merge the preferred one.

**Comment by ArcLumin** [comment]
The anti-stuns PR also edits this shield

**Comment by Swagile** [comment]
either mcbawg removes his edit or this gets closed

till either or happens im neutral on this pr

**Comment by SpankMasterson** [comment]
I +1 Mcbaws' or dong's, whichever is chosen

**Comment by McBawbaggings** [comment]
This effectively got merged.

**Pull request closed without merging**